src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/IdentifiersShouldDifferByMoreThanCase.cs

```diff
@@ -125,86 +126,133 @@ private static void CheckNamespaceMembers(IEnumerable<INamespaceSymbol> namespac
 
         private static void CheckTypeMembers(IEnumerable<ISymbol> members, Action<Diagnostic> addDiagnostic)
         {
-            // Remove constructors, indexers, operators and destructors for name check
-            IEnumerable<ISymbol> membersForNameCheck = members.Where(item => !item.IsConstructor() && !item.IsDestructor() && !item.IsIndexer() && !item.IsUserDefinedOperator());
-            if (membersForNameCheck.Any())
+            // If there is only one member, then return
+            if (!members.Skip(1).Any())
             {
-                CheckMemberNames(membersForNameCheck, addDiagnostic);
+                return;
             }
-        }
 
-        private static void CheckParameterMembers(IEnumerable<ISymbol> members, Action<Diagnostic> addDiagnostic)
-        {
-            IEnumerable<ISymbol> violatingMembers = members
-                .Where(item => item.ContainingType.DelegateInvokeMethod == null && HasViolatingParameters(item));
-
-            IEnumerable<ISymbol> violatingDelegates = members.Select(item =>
+            using var overloadsToSkip = PooledHashSet<ISymbol>.GetInstance();
+            var membersByName = PooledDictionary<string, PooledHashSet<ISymbol>>.GetInstance(StringComparer.OrdinalIgnoreCase);
```

Comment:
>var membersByName [](start = 12, length = 17)

using ?

Reply:
Haha, defeated the whole purpose of my refactoring :) Will fix...

Reply:
It's not a LINQ allocation...  😁